tasks/minsize/minsize.snap

```diff
@@ -1,25 +1,25 @@
 Original   -> Minified   -> Gzip       Brotli    
 72.14 kB   -> 24.36 kB   -> 8.68 kB    7.64 kB    react.development.js
 
-173.90 kB  -> 61.97 kB   -> 19.46 kB   17.76 kB   moment.js
+173.90 kB  -> 61.89 kB   -> 19.47 kB   17.70 kB   moment.js
```

Comment:
Why the Gzip becomes larger? I don't really know.

Reply:
This is fine, we want the overall size to go smaller first.